kivy/uix/modalview.py

```diff
@@ -70,19 +72,20 @@ def my_callback(instance):
     keyboard if the :attr:`ModalView.auto_dismiss` property is True (the
     default).
 
-'''
+"""
 
 __all__ = ('ModalView', )
 
-from kivy.logger import Logger
 from kivy.animation import Animation
+from kivy.core.window import Window
```

Comment:
```suggestion
```
I glossed over this before, but generally we try not to import `Window` at the module level because that causes a Window to be created upon import. Sometimes it's unavoidable and I know that importing kivy will already generally create the window, but it's better to reduce this for now until we can figure out how to resolve this issue of Kivy creating a Window when Window is imported.